### DISCUSSION TITLE
The time zone in Qazaqstan has changed

### DISCUSSION THREAD

**Discussion by serjan-nasredin** [community/community#112439] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

![image](https://github.com/community/community/assets/67647968/9c034d97-d9b6-4ce0-8779-978c2f3fb9a7)
The time zone in Qazaqstan has switched to a single time (GTM+5). In this regard, I want to ask: when can I change my time zone to Astana/Almaty (GTM+5), and not Tashkent (GTM+5)?

**Comment by kacphur** [comment]
Check out your device settings and see if there is an option available!

**Reply by serjan-nasredin** [reply]
Nope
